Assets/Scripts/PlayerController.cs

```diff
@@ -297,7 +300,22 @@ void DisplayInteractionPrompts()
             bp2.hidePrompt();
         }
     }
-
+    void showInteractionIcon()
```

Comment:
Can you follow the style convention for naming functions. It should be `ShowInteractionIcon`